src/components/__tests__/IspSelectWithIncidents-test.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import chai, { expect } from 'chai';
+
+import chaiEnzyme from 'chai-enzyme';
+import { IspSelectWithIncidents } from 'components';
+
+chai.use(chaiEnzyme());
+
+describe('components', () => {
+  describe('IspSelectWithIncidents', () => {
+    it('selecting multiple ISPs', () => {
+
+      let selected = [];
+
+      const mockChange = function(ispId) { selected.push(ispId); };
+
+      // Sample data for this test
+      const isps = {
+        AS10774x: { client_asn_name: 'AT&T', client_asn_number: 'AS10774x' },
+        AS10796x: { client_asn_name: 'Time Warner Cable', client_asn_number: 'AS10796x' },
+        AS11486x: { client_asn_name: 'Verizon', client_asn_number: 'AS11486x' },
+        AS10507: { client_asn_name: 'Sprint Personal Communications Systems', client_asn_number: 'AS10507' },
+      };
+        
+      const incidentData = {};
+
+      const onChangeIncidentASN = function(){return;};
+
+      const wrapper = mount(
+        <IspSelectWithIncidents
+          incidentData={incidentData}
+          isps={isps}
+          onChangeIncidentASN={onChangeIncidentASN}
+          selected={selected}
+          onChange={mockChange}
+        />
+      );
+
+      // Call select function on AT&T
+      wrapper.instance().onAdd({client_asn_name: 'AT&T', client_asn_number: 'AS10774x' });
+
+      // Make sure that an item has been added to parent's 'selected' member
+      expect(selected).to.have.lengthOf(1);
+
+      // Add Verison
+      wrapper.instance().onAdd({client_asn_name: 'Verison', client_asn_number: 'AS11486x' });
```

Comment:
instead of calling wrapper.instance() twice (line 41), you can define a variable 'const instance = wrapper.instance();' and then just use that variable